File: lib/models/feedback.dart
class Feedback {
  final String sender;
  final String subject;
  final String messageBody;

  Feedback({required this.sender, required this.subject, required this.messageBody});

  Map<String, dynamic> toJson() {
    return {
      'sender': sender,
      'subject': subject,
      'messageBody': messageBody,
    };
  }
}

File: backend -nodejs--chat-feedback-app/models/feedback.js
const mongoose = require('mongoose');

const FeedbackSchema = new mongoose.Schema({
  sender: String,
  subject: String,
  messageBody: String,
  timestamp: {
    type: Date,
    default: Date.now
  }
});

module.exports = mongoose.model('Feedback', FeedbackSchema);
